testdata/testoutput10

```diff
@@ -1603,14 +1618,17 @@ No match
 
 /(?'ABáC6789012345678901234567890123012345678901234567890123456789AB012345678901234567890123456789AB012345678901234567890123456789AB'...)/utf
 Failed: error 148 at offset 132: subpattern name is too long (maximum 128 code units)
+        here: ...23456789AB |<--| '...)
 
 /(?'ABZC6789012345678901234567890123012345678901234567890123456789AB012345678901234567890123456789AB012345678901234567890123456789AB'...)/utf
 
 /(?(n/utf
 Failed: error 142 at offset 4: syntax error in subpattern name (missing terminator?)
+        here: (?(n |<--|
```

Comment:
I often see using the caret (^)
```
/(?(n/utf
     ^
Failed: error 142 at offset 4: syntax error in subpattern name (missing terminator?)
```

Reply:
That would be a better user interface. I did consider it, and even attempted it. However, editors and terminals differ in how they render Unicode combining characters with multiple codepoints, for example characters with decomposing accents. Rendering the caret on the line below would be less precise for the purpose of a testing tool.

Reply:
Ok